# Names of each machine type.
machine_type_names:
  - "Loading"
  - "Mega"
  - "Vinyl"
  - "Hem"
  - "Tack"
  - "Weld"
  - "Grommet"
  - "Inspection"

# Abbreviations to use when drawing schedules.
machine_type_abvs:
  - "L"
  - "MS"
  - "V"
  - "H"
  - "T"
  - "RF"
  - "G"
  - "I"

# Number of each machine type.
machine_type_nums:
  - 1
  - 1
  - 1
  - 1
  - 1
  - 1
  - 1
  - 1
# machine_type_nums:
#   - 2
#   - 2
#   - 2
#   - 4
#   - 2
#   - 4
#   - 4
#   - 1